packages/connector-sendgrid-mail/src/types.ts

```diff
@@ -0,0 +1,119 @@
+import { Nullable } from '@silverhand/essentials';
+import { z } from 'zod';
+
+/**
+ * @doc https://docs.sendgrid.com/api-reference/mail-send/mail-send#body
+ */
+
+export enum ContextType {
+  'TEXT' = 'text/plain',
+  'HTML' = 'text/html',
+}
+
+export type EmailData = {
+  name?: string;
+  email: string;
+};
+
+export type Personalization = {
+  to: EmailData[];
```

Comment:
Should `to` be `EmailData | EmailData[]` as well?

Reply:
Good call. It shows EmailData | EmailData[] in the doc, but threw an error when requesting with the EmailData type.

Reply:
[facepalm]..